server.py

```diff
@@ -0,0 +1,85 @@
+from __future__ import unicode_literals
+import socket
+
+
+_CRLF = b'\r\n'
+_GET = b'GET'
+_PROTOCOL = b'HTTP/1.1'
+_HOST_PREFIX = b'Host:'
+_WS = b' '
```

Comment:
The initial underscores here look a bit odd to the trained Python eye.  I'd leave 'em out.